### ISSUE TITLE
Add an option  to remove original video files after transcoding.

### ISSUE THREAD

**Issue by kyrahabattoir** [Chocobozzz/PeerTube#3559]
**Describe the problem to be solved**

Instances like mine just can't handle the disk space required to store full size videos, whether they are uploaded as 1080p or recovered from youtube, so I typically have 360p transcoding enabled, but there appears to be no way to auto-remove videos that do not match any of the transcoding options.

**Describe the solution you would like:**

Essentially I'd like a configuration option to delete the source file after transcoding, if it doesn't match any of the transcoding targets, so if i upload a 1080p video and the instance is confugured for 720p transcoding, once the transcoding job is done, the 1080p should be removed.

**Comment by Chocobozzz** [comment]
Thanks for reporting this issue, but this is a duplicate of https://github.com/Chocobozzz/PeerTube/issues/1030

**Comment by maxlinux2000** [comment]
Hello, I have written a simple wrapper that allows you to import videos or entire channels from YouTube at the resolution you want, by default it is 360p

You can download it from this article below in Spanish. The script is in English and is self explanatory.
You can use an automatic translator to understand it better

https://isla1.tormentasolar.win/importar-canales-enteros-en-peertube/

regards
MaX
